backend/tests/MembersDao.test.ts

```diff
@@ -26,8 +31,8 @@ test('Add new member', () => {
 });
 
 test('Get member from past semester', () =>
-  MembersDao.getCurrentOrPastMemberByEmail(jaggerData.email).then((pastMember) =>
-    expect(pastMember).toEqual(jaggerData)
+  MembersDao.getCurrentOrPastMemberByEmail(jaggerData.email).then(
+    (pastMember) => expect(pastMember).toEqual(jaggerData) // cannot deprecate jaggerData yet
```

Comment:
This is the only place left that uses the hardcoded mock user `jaggerData`. This test for the `getCurrentOrPastMemberByEmail` function relies on accessing a json archive of previous members. Is it worth it to automatically add a generated user to the archive and remove it after the test runs, or is it fine to keep the hardcoded mock data in this case?